Sources/CardPayments/CardClient.swift

```diff
@@ -176,13 +176,13 @@ public class CardClient: NSObject {
             context: self,
             sessionDidDisplay: { [weak self] didDisplay in
                 if didDisplay {
-                    // should we send different analytics for vault 3DS?
+                    // TODO: analytics for card vault
                     self?.analyticsService?.sendEvent("card-payments:3ds:challenge-presentation:succeeded")
```

Comment:
Should I send same 3DS analytics for vault without purchase flow? 
For other analytics events, should I do them here or create another ticket?

Reply:
It's a good practice to leave a TODO with the JIRA ticket # listed as a comment